content/300-guides/050-database/820-multi-schema.mdx

```diff
@@ -0,0 +1,101 @@
+---
+title: How to use Prisma with multiple database schemas
+metaDescription: How to use Prisma with multiple database schemas
+tocDepth: 2
+---
+
+<TopBlock>
+
+<Admonition type="warning">
+
+Multiple database schema support is currently a very early, experimental preview feature. Introspection and applying your Prisma schema to your database with `migrate dev` and `db push` are currently not supported. If you rely on them, please do not try this preview feature in your main branch.
```

Comment:
I got this from the issue comment, but does it really mean 'in your main branch' or something more like 'in production'?

Reply:
I clarified the sentence in the issue comment a bit (Frederik added that later):

> Migrate and introspection are currently not supported with multiple schemas. If you rely on them, please do not try this preview feature in the main branch of your project or in production.

I think you could use similar wording if you want. Frederik mostly wanted to avoid people trying to use this, and then being disappointed how it makes their migration history or main deployments harder to maintain and update.

Additionally: The explicit listing of `migrate dev` and `db push` excludes all other ways to migrate your database like `migrate deploy`.

Reply:
Thanks, I've added in the new wording. 

I've also changed it to 'with Prisma Migrate and `db push`', does that work?

Reply:
Yes, that sounds better.